microsoftgraph/microsoft-graph-explorer-v4
# Feature: Surface documentation links

## Overview

Walks through the sample queries and resource explorer to get documentation links for the URL on the Query Input.

When available, changes the more info icon to be a documentation icon that allows a user to open the related link in a new tab

Closes #1854

## Testing Instructions

* Type in a URL
* Notice that the icon changes from disabled to enabled
* Click on the documentation icon
* A tooltip opens with the statement that documentation is available for the URL
* Click learn more and see the documentation on a new tab
